std_msgs/UInt16 req_id
std_msgs/UInt16 ttl
std_msgs/String destination
std_msgs/ByteMultiArray data
